pennylane/measurements.py

```diff
@@ -428,17 +441,20 @@ def _queue_category(self):
     @property
     def hash(self):
         """int: returns an integer hash uniquely representing the measurement process"""
+        wires_list = [self.wires] if not isinstance(self.wires, list) else self.wires
+        wires_list = [wire for wires in wires_list for wire in wires.tolist()]
```

Comment:
Is it necessary given the fact that `self.wires` cannot return a list of wires as you used `all_wires`?

Reply:
You're right, it doesn't look like it's necessary.

Reply:
Removed these two lines.